examples/tests/i2c/i2c_master_slave_ping_pong/main.c

```diff
@@ -99,6 +99,6 @@ int main(void) {
 
   int j;
   for (j = 0; j < nbuttons; j++) {
-    TOCK_EXPECT(TOCK_SUCCESS, button_enable_interrupt(j));
+    TOCK_EXPECT(TOCK_SUCCESS, button_enable_interrupt(j).type);
```

Comment:
```suggestion
    TOCK_EXPECT(TOCK_SYSCALL_SUCCESS, button_enable_interrupt(j).type);
```

I have made this mistake several times already